lib/screens/lobby.dart

```diff
@@ -20,14 +24,108 @@ class LobbyScreen extends StatelessWidget {
               };
 
               context.read<NavigationService>().showConfirmationDialog(
-                  onConfirmed: signOut,
-                  confirmText: 'Sign Out',
-                  cancelText: 'Return');
+                    onConfirmed: signOut,
+                    confirmText: 'Sign Out',
+                    cancelText: 'Return',
+                  );
             },
             child: Text('log out'),
           )
         ],
       ),
+      body: Container(
+        child: StreamBuilder<List<Game>>(
+            stream: context.watch<DatabaseService>().streamOfCreatedGames,
+            builder: (context, snapshot) {
+              if (snapshot.data == null) {
+                print('still null');
+                return Container();
+              } else {
+                print(' stream builder working');
+                return Padding(
+                  padding: const EdgeInsets.fromLTRB(0, 10, 0, 0),
+                  child: ListView(
+                    children: <Widget>[
+                      for (var game in snapshot.data) GameTile(game)
```

Comment:
love the use of "collection for"!

Reply:
another way to achieve the same result would be ListView.builder - did you consider using that instead of the default constructor? if the list of games could potentially get quite long it may be the better option as using the default constructor builds all the list items whereas ListView.builder only builds what is on screen